src/mscorlib/src/System/Char.cs

```diff
@@ -234,7 +234,7 @@ public bool Equals(Char obj)
           if (IsLatin1(c)) {
               if (IsAscii(c)) {
```

Comment:
@JonHanna True; most chars inputted into this method are going to be all-ASCII though, so I don't think we should penalize for the common case by having two branch checks before the return. In fact, the method could be written as

```cs
if ((c | '\u0020').IsBetween('a', 'z'))
    return true;
// Handle Latin1, non-Latin1...
```

to avoid the `IsAscii` check altogether.